### DISCUSSION TITLE
edge dot . deprecated error

### DISCUSSION THREAD

**Discussion by dmeinke-etc** [adonisjs/core#3583] [Help] [answered]
I get this error in my logs 

```
(node:150) edge: dot "." based path seperators are depreciated. We recommend using "/" instead: DeprecationWarning
```

Using node:16-alpine3.12 in my container

Ideas on what I can do to stop this warning?

**Comment by RomainLanz** [comment, marked as answer]
By not using a deprecated syntax.

**Reply by dmeinke-etc** [reply]
Yeah, that seems like a logical solution. The only place I can see where I might be using a dot in a path is in my routes. But thats a method call and not a path. 
```
Route.group(() => {
  Route.get('/tasks', 'TasksController.index')
  Route.get('/tasks_add', 'TasksController.add')
  Route.post('/tasks', 'TasksController.create')
  Route.get('/tasks/:id/edit', 'TasksController.edit')
  Route.patch('tasks/:id', 'TasksController.update')
}).middleware('isAdmin')
```
All of my renders are normal
```
 return view.render('maintenance/task', { tasks })
```

So I am unsure of what path separators it is referring to that are depricated?

**Reply by RomainLanz** [reply]
Are you sure inside your template you are not referring to `layouts`, `includes`, or components with `.`?

**Reply by dmeinke-etc** [reply]
That appears to be the problem. I will go through my templates and clean those up.

**Reply by oladejihenry** [reply]
I think Adonis controller docs should be updated so that new devs coming to Adonis js won't be confused when they see the error cause ```('posts.index')``` should be replace with ```('posts/index')```.

Thanks

**Reply by thetutlage** [reply]
@oladejihenry Can you point us to the doc which uses the dot syntax? Or maybe create a PR to fix it :)
